README.md

```diff
@@ -19,5 +19,5 @@ For access to the DBC file which defines the CAN interface for the PACMod, see t
 | Ford Ranger 2019+ | ROS | PACMod3 | [PACMod3 Driver (DBC 11)](https://github.com/astuff/pacmod3/tree/devel/dbc_11) |
 | Polaris Ranger X900 | ROS | PACMod3 | [PACMod3 Driver (DBC 11)](https://github.com/astuff/pacmod3/tree/devel/dbc_11) |
 | Toyota Minivan 2019+ | ROS | PACMod3 | [PACMod3 Driver (DBC 11)](https://github.com/astuff/pacmod3/tree/devel/dbc_11) |
-
+| VEHICLE_HCV | ROS | PACMod3 | [PACMod3 Driver with message migration](https://github.com/astuff/pacmod3/tree/maint/pacmod_msg_migration-vehicle_hcv) |
```

Comment:
All the other vehicles above have the manufacturer and model year, but this entry seems more like a constant used in the code. What exactly is `VEHICLE_HCV`?